x/gamm/pool-models/balancer/amm_test.go

```diff
@@ -1292,15 +1294,32 @@ func TestRandomizedJoinPoolExitPoolInvariants(t *testing.T) {
 	rng := rand.NewSource(now)
 	t.Logf("Using random source of %d\n", now)
 
-	// generate test case with randomized initial assets and join/exit ratio
-	newCase := func() (tc *testCase) {
+	newBaseCase := func() (tc *testCase) {
 		tc = new(testCase)
-		tc.initialTokensDenomIn = rng.Int63() % 1_000_000
-		tc.initialTokensDenomOut = rng.Int63() % 1_000_000
+		tc.initialTokensDenomOne = rng.Int63() % 1_000_000
+		tc.initialTokensDenomTwo = rng.Int63() % 1_000_000
+		return tc
+	}
+
+	// generate test case with randomized initial assets and join/exit ratio
+	// where ratios of tokensIn to liquidity in pool is the same.
+	newEqualRatioCase := func() (tc *testCase) {
+		tc = newBaseCase()
 
 		// 1%~100% of initial assets
-		tc.percentRatio = rng.Int63()%100 + 1
+		tc.percentRatioOne = rng.Int63()%100 + 1
+		tc.percentRatioTwo = tc.initialTokensDenomOne
```

Comment:
Should be ?
```suggestion
                percentRatio := rng.Int63()%100 + 1
		tc.percentRatioOne = percentRatio
		tc.percentRatioTwo = percentRatio
```

Reply:
Nice catch, yes, long day yesterday :tired_face: